global/sass-config/_mixins.sass

```diff
@@ -44,4 +44,8 @@
 
 // Standard border radius
 =border-radius()
-  border-radius: 5px
\ No newline at end of file
+  border-radius: 5px
+  
+// Slide
+=slide($percentage)
```

Comment:
Currently named slide not slide-up

Reply:
I did this intentionally so it could also be used to slide down if you input a positive value. Should I change it?